### PR TITLE
Fix S3 Configuration

### PR DESCRIPTION
**Pull request by dailos2coders**
Mattermost were not using the env variables for S3 correctly. This commits fixes the problem

### CODE REVIEW

**Comment by hanzei** [comment]
@jwilander Pinging you because your created this repo. Who would be a good one to review this PR?

**Comment by cpanato** [comment]
@dailos2coders thanks for this PR, can you please update your PR with latest master?

**Comment by dailos2coders** [comment]
I have merged master, and upgraded Mattermost to latest version

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# Mattermost Heroku
+# Deploy Mattermost Team or Enterprise Edition to Heroku
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+Updated version of Mattermost to deploy in HEROKU. Updated by @2CodersTeam
```
Comment (cpanato) [inline README.md]: I think you dont need to set this `Updated by @2CodersTeam`

any comments @jwilander ?

Comment (dailos2coders) [reply README.md]: no problem... just removed it

**Review by cpanato — APPROVED** [review]
lgtm, thanks for that!

**Comment by cpanato** [comment]
@jwilander please take a look when you have some time

**Review by jwilander — APPROVED** [review]

**Pull request merged**